src/heroku-keepalive.coffee

```diff
@@ -23,11 +23,11 @@
 #   Josh Nichols <technicalpickles@github.com>
 
 module.exports = (robot) ->
-  wakeUpTime = (process.env.HUBOT_HEROKU_WAKEUP_TIME or '6:00').split(':')
-  sleepTime = (process.env.HUBOT_HEROKU_SLEEP_TIME or '22:00').split(':')
+  wakeUpTime = (process.env.HUBOT_HEROKU_WAKEUP_TIME or '6:00').split(':').map (i) -> i * 1
```

Comment:
Is this just converting them to integers? Might be able to use `parseInt` instead.